docs/HelpWantedProcess.md

```diff
@@ -0,0 +1,13 @@
+This document describes the process that the team uses for identifying candidate issues for community members to tackle.
+
+## Process
+
+This process (we'll refer to it as `help-wanted` process going forward) triggers from the moment, when the engineering team decides that a particular issue may be a good fit for a community member to tackle. Below are the stages that issue will go through, as part of this process:
```

Comment:
Also should there be a process that a community member offer to do an issue even if the issue does not have help-wanted label? I myself am doing this by commenting on the issue and wait for some one on the team to confirm it.

Reply:
There is no intention to break things which work today by any means. If commenting on issues without `help-wanted` label and getting confirmation from the team member that it is a good approach to take works for you, then it will continue to work 👍